go-controller/pkg/ovn/master.go

```diff
@@ -908,5 +950,17 @@ func (oc *Controller) syncNodes(nodes []interface{}) {
 		if err := oc.deleteNode(nodeName, nodeSubnets.hostSubnet, nodeSubnets.joinSubnet); err != nil {
 			klog.Error(err)
 		}
+		//remove the node from the chassis map so we don't delete it twice
+		delete(chassisMap, nodeName)
+	}
+
+	for nodeName, chassisName := range chassisMap {
```

Comment:
@girishmg this should take care of deleting the chassis even if it gets re-added.

Reply:
Nevermind, looks like `syncNodes` is only called once when ovnkube-master starts.